### PR TITLE
Circle bonus icons

### PR DESCRIPTION
**Pull request by normaljosh**
Shell script to extract circles and numbers as requested in #172. 

I added both the circles and just the numbers into the spritesheet, changes are in circle_icons.sh

### CODE REVIEW

**Comment by micahstairs** [comment]
Thanks! I'll take a closer look at this later, since there is higher priority stuff for me to review. In the meantime, there's another ImageMagick task if you're interested: https://github.com/micahstairs/bga-innovation/issues/199.

File: misc/imagemagick_commands/circle_icons.sh [hunk 1]
```diff
@@ -6,29 +6,77 @@ mkdir temp
...
 
 echo "Extracting echoes circle icons..."
 
-# Age 1 examples
-extract_circle "002" "$BOTTOM_RIGHT" "000"
-extract_circle "003" "$BOTTOM_LEFT"  "001"
-extract_circle "012" "$BOTTOM_CENTER"  "002"
-extract_circle "011" "$TOP_LEFT"  "003"
+# Extract blue, red, green, yellow, purple bonus circles
+extract_circle "013" "$BOTTOM_LEFT" "1"
```
Comment (micahstairs) [inline misc/imagemagick_commands/circle_icons.sh:44]: Is it easy to cover up the numbers in these 5 circles?

File: misc/imagemagick_commands/circle_icons.sh [hunk 2]
```diff
@@ -6,29 +6,77 @@ mkdir temp
...
+extract_circle "059" "$BOTTOM_LEFT"  "3"
+extract_circle "003" "$BOTTOM_LEFT"  "4"
+extract_circle "022" "$BOTTOM_LEFT"  "5"
+
+# Extract numbers 1-11 
+extract_ellipse "002" "$BOTTOM_RIGHT_ELLIPSE" "1"
+extract_ellipse "003" "$BOTTOM_LEFT_ELLIPSE"  "2"
+extract_ellipse "012" "$BOTTOM_CENTER_ELLIPSE"  "3"
+extract_ellipse "034" "$BOTTOM_CENTER_ELLIPSE"  "4"
+extract_ellipse "042" "$BOTTOM_LEFT_ELLIPSE"  "5"
```
Comment (micahstairs) [inline misc/imagemagick_commands/circle_icons.sh:55]: The 5 looks a bit off compared to the other ellipses. Is the source image a few pixels off?

**Review by micahstairs — CHANGES_REQUESTED** [review]
Thanks for tackling this!

File: misc/imagemagick_commands/circle_icons.sh (hunk 2 above)
Comment (normaljosh) [reply misc/imagemagick_commands/circle_icons.sh:55]: Yeah, seemed like that position was a little different across the cards. I went with bottom right where possible and the numbers seem more centered now.

**Comment by normaljosh** [comment]
@micahstairs sorry for the delay on this one, I've centered the images and covered up the numbers.

File: misc/imagemagick_commands/circle_icons.sh (hunk 1 above)
Comment (micahstairs) [reply misc/imagemagick_commands/circle_icons.sh:45]: Amazing, thanks!

**Comment by micahstairs** [comment]
Thanks for finishing this up! No worries about the delay, this is great timing since we are shifting some of our attention towards Echoes/Cities now.

**Review by micahstairs — APPROVED** [review]

**Pull request merged**